internal/cmd/config/root.go

```diff
@@ -110,6 +110,9 @@ func printConfig(config config.UnvalidatedConfig) {
 	print.Entry("GitLab connector type", format.OptionalStringerSetting(config.NormalConfig.GitLabConnectorType))
 	print.Entry("GitLab token", format.OptionalStringerSetting(config.NormalConfig.GitLabToken))
 	fmt.Println()
+	print.Header("Proposals")
```

Comment:
Let's stick with the verb-based heading titles where possible.

```suggestion
	print.Header("Propose")
```

Reply:
https://github.com/git-town/git-town/pull/5610/commits/79423b27f74651b1dec7468ca59113d5ea95ffd0